### ISSUE TITLE
Sounds do not play through bluetooth connected headphones

### ISSUE THREAD

**Issue by chrisgarrity** [LLK/scratchjr#427]
### Expected Behavior

Sounds, both recorded and UI effects, should playback through a bluetooth headset when connected to the device.

### Actual Behavior

On iPads the sounds continue to play back through the speakers on the iPad even if other apps (e.g. Safari) play back through the headphones. 

### Steps to Reproduce

1. Pair bluetooth headphones with the iPad
2. Run ScratchJr, the  UI sounds play through the speakers
3. Try recording a sound, the sound plays back through the speakers

Note: it appears that the audioSession needs to be set up with bluetooth enabled. I tried changing every instance of
```
AVAudioSession *audioSession = [AVAudioSession sharedInstance];
    [audioSession setCategory:AVAudioSessionCategoryPlayAndRecord error:nil];
```
to
```
AVAudioSession *audioSession = [AVAudioSession sharedInstance];
    [audioSession setCategory:AVAudioSessionCategoryPlayAndRecord withOptions:AVAudioSessionCategoryOptionAllowBluetooth error:nil];
```
Sounds started playing through my headset. However, as each session is stopped, I would get the message "call ended" in my headset. So this isn't a complete solution.

### Device and Operating System
iPad mini 4, iOS 13.5.1

**Comment by deepankarmalhan** [comment]
Also check on Android

**Comment by deepankarmalhan** [comment]
Released in 1.4.14
